types/src/gens.rs

```diff
@@ -1108,60 +1108,80 @@ pub fn transaction_invocation_target_arb() -> impl Strategy<Value = TransactionI
 pub fn stored_transaction_target() -> impl Strategy<Value = TransactionTarget> {
     (
         transaction_invocation_target_arb(),
-        transaction_runtime_arb(),
-        any::<u64>(),
+        transaction_stored_runtime_params_arb(),
     )
-        .prop_map(|(target, runtime, transferred_value)| {
-            TransactionTarget::new_stored(target, runtime, transferred_value)
-        })
+        .prop_map(|(id, runtime)| TransactionTarget::Stored { id, runtime })
+}
+
+fn transferred_value_arb() -> impl Strategy<Value = u64> {
+    any::<u64>()
```

Comment:
I wonder if having a random value here makes any sense. IMHO we should have [0; u64::MAX] to prove that it handles both ends of the range? maybe MAX/2 if we are super zealous?

Reply:
I'd keep it consistent with other places where we use `any()` but could limit the values to those that only make sense in given context